sdk/deviceupdate/iot-device-update-rest/test/public/utils/env.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as dotenv from "dotenv";
+
+dotenv.config();
```

Comment:
I'm confused why this is necessary, I think the recorder does this for you? /cc @HarshaNalluru

Reply:
I think it works. After my test, when I create `.env` file in the package folder and set environment variables in it, I can get it in variable `env` of test recorder.

Reply:
If you delete this file, does it continue to work?

Reply:
No. If this file is deleted, we cannot get the env variable from `.env` file in creating client.